ui/src/components/Preview/Overview.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import styles from "./Preview.module.css";
+import { DashboardMetricOverview } from "../../api/Api";
+import { IconButton, Tab, Tabs } from "@mui/material";
+import PieChart from "../Piechart";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import { BootstrapTooltip } from "../BootstrapTooltip";
+import { docRedirects } from "../../types/component.type";
+
+interface IOverviewProps {
+  dashboard: DashboardMetricOverview;
+  width: number;
+}
+
+const Overview: React.FC<IOverviewProps> = ({ dashboard, width }) => {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  interface ITabPanelProps {
+    children?: React.ReactNode;
+    index: number;
+    value: number;
+  }
+
+  const TabPanel: React.FC<ITabPanelProps> = ({ children, index, value }) => {
+    return (
+      <div
+        role="tabpanel"
+        hidden={value !== index}
+        id={`vertical-tabpanel-${index}`}
+        aria-labelledby={`vertical-tab-${index}`}
+      >
+        {value === index && (
+          <div>
+            <p>{children}</p>
+          </div>
+        )}
+      </div>
+    );
+  };
+
+  function a11yProps(index: number) {
+    return {
+      id: `vertical-tab-${index}`,
+      "aria-controls": `vertical-tabpanel-${index}`,
+    };
+  }
+
+  return (
+    <div className={styles.card}>
+      <div className={styles.cardInfo}>
+        <h1 className={styles.header}>Overview</h1>
+        <p className={styles.description}>
+          Key metrics providing insights into the integrity and uniqueness of
+          your data.
+        </p>
+      </div>
+      <div className={styles.snapscoreMain}>
+        <div className={styles.verticalTabs}>
+          <Tabs
+            orientation="vertical"
+            variant="scrollable"
+            indicatorColor="primary"
+            value={value}
+            textColor="inherit"
+            onChange={handleChange}
+            TabIndicatorProps={{
+              style: {
```

Comment:
@driptanil Please find a way to pass a classname rather than putting CSS in props, and if it's not there then let it be!